### PR TITLE
Homepage docs

### PR DESCRIPTION
**Pull request by ross-pfahler**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [Issue](https://github.com/adobe-private/react-spectrum-v3/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Team:

React Spectrum

### CODE REVIEW

File: packages/dev/docs/pages/index.mdx [hunk 1]
```diff
@@ -10,8 +10,37 @@ governing permissions and limitations under the License. -->
...
+
 # React Spectrum
 
-- [React Spectrum](./react-spectrum/index.html)
-- [React Aria](./react-aria/index.html)
-- [React Stately](./react-stately/index.html)
+A collection of libraries and tools that help you build adaptive, accessible, and robust user experiences.
+
+<Image type="hero">
+  <img srcset={`${heroNarrow} 600w,
```
Comment (snowystinger) [inline packages/dev/docs/pages/index.mdx:22]: hrm... these are coming up as broken links

Comment (ross-pfahler) [reply packages/dev/docs/pages/index.mdx:22]: @devongovett is looking into it

File: packages/dev/docs/src/docs.css [hunk 2]
```diff
@@ -30,6 +30,46 @@ html, body {
...
+
+  div {
+    box-sizing: border-box;
+    padding: 0 16px;
+  }
+
+  @media (min-width: 1020px) {
+    div:first-child:nth-last-child(3),
+    div:first-child:nth-last-child(3) ~ div {
+      width: 33.3333%;
```
Comment (snowystinger) [inline packages/dev/docs/src/docs.css:63]: 👀 
wha? why?

Comment (ross-pfahler) [reply packages/dev/docs/src/docs.css:63]: magic! if 3 items then width is 33%, otherwise width is different (see later code)

**Pull request merged**